opal/util/arch.h

```diff
@@ -178,7 +178,7 @@
 **   bits 1 & 2: length of long double: 00=64, 01=96,10 = 128
 **   bits 3 & 4: no. of rel. bits in the exponent: 00 = 10, 01 = 14)
 **   bits 5 - 7: no. of bits of mantisse ( 000 = 53,  001 = 64, 010 = 105,
```

Comment:
@markalle Your next line change is good, but to keep columns aligned (notice the two spaces after `53,`), I think you fix this one too:
```suggestion
**   bits 5 - 7: no. of bits of mantisse ( 000 = 53,  001 = 64,  010 = 105,
```

Reply:
making the change, thx